launcher/image/cloudbuild.yaml

```diff
@@ -0,0 +1,48 @@
+substitutions:
+  '_BASE_IMAGE': ''
+  '_BASE_IMAGE_PROJECT': 'cos-cloud'
+  '_OUTPUT_IMAGE_SUFFIX': ''
+  '_IMAGE_ENV': 'debug'
```

Comment:
Thoughts on changing this to "_IS_DEBUG" (boolean) and doing away with the concept of "hardened" (like how we have for our stable releases)?

Reply:
I think it is better to be explicit about the value here, because it will be passed around different cloudbuild/scripts